.pipelines/publish-to-pypi.yml

```diff
@@ -0,0 +1,33 @@
+variables:
+  - group: Presidio-V2-CI
+steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.8'
+  - task: Bash@3
+    displayName: 'Package Wheel'
+    inputs:
+      targetType: 'inline'
+      script: |
+        set -eux  # fail on error
+        pip install wheel
+        python setup.py bdist_wheel
+  - task: Bash@3
+    displayName: 'Install twine'
+    inputs:
+      targetType: 'inline'
+      script: |
+        set -eux  # fail on error
+        pip install twine
+  - task: TwineAuthenticate@1
+    displayName: 'Authenticate with PyPi'
+    inputs:
+
+      pythonUploadServiceConnection: Pypi-Presidio
+  - task: Bash@3
+    displayName: 'Upload Wheel to PyPi'
+    inputs:
+      targetType: 'inline'
+      script: |
+        set -eux  # fail on error
+        twine upload -r presidio --verbose --config-file $(PYPIRC_PATH) --skip-existing dist/*
```

Comment:
Should this be `twine upload -r presidio-evaluator`?

Reply:
That's what I though originally, but -r is the repository, and we have the PyPI setup in AzDO where under the repository "presidio" we the configuration and credentials.